packages/core/src/utils.ts

```diff
@@ -1158,8 +1156,12 @@ export const isLocalNetwork = async (
     return false
   }
 
-  if (await isHardhatFork(provider)) {
-    return false
+  try {
+    if (await isHardhatFork(provider)) {
+      return false
+    }
+  } catch (e) {
+    return true
```

Comment:
Just curious, what situation is the try/catch necessary? The first try/catch in this function should prevent `isHardhatFork` function from being called on live networks

Reply:
`isHardhatFork` throws an error when run against anvil, and `isLocalNetwork` is called from the initializeChugSplash function which I'm using here to deploy the contracts against anvil. So this try-catch, makes it assume anvil is a local network no matter what. I did it this way b/c I didn't want to refactor the `initializeChugSplash` function to avoid using `isLocalNetwork` against anvil since we only need to use it until we have the raw transaction cheat code.